s2i/bin/run

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+source /opt/app-root/etc/generate_container_user
+
+set -e
+
+. ${VARNISH_CONFIGURATION_PATH}/varnish.params
+
+if grep "NAME=Fedora" /etc/os-release; then
+	exec varnishd \
+		-F \
+		-f $VARNISH_VCL_CONF \
+		-a :8080 \
+		-T 127.0.0.1:8081 \
+		-t $VARNISH_TTL \
+		-S $VARNISH_SECRET_FILE \
+		-s $VARNISH_STORAGE \
+		$DAEMON_OPTS
+else
+	exec varnishd \
+		-F \
+		-f $VARNISH_VCL_CONF \
+		-a :8080,:8443 \
+		-T 127.0.0.1:8081 \
+		-t $VARNISH_TTL \
+		-u default \
```

Comment:
`-j "unix,user=default,ccgroup=root"` can be used for newer varnishd.

So IMPOV it would be more readable to have if/else for varnish version -> set right parameters there (`-a X,Y` vs `-a X -a Y` or `-u`,`-g` vs `-j XY`). And then have only one `exec varnishd ...`

Reply:
Hmmm, docs seem to be misleading:

```
[INFO] Waiting for application container to start...

Can't read /etc/scl/conf/rh-varnish4, rh-varnish4 is probably not installed.
Error: Too many arguments (unix,user=varnish,ccgroup=root...)
(-? gives usage)
```

Reply:
(never mind, I'm a moron, the problem was elsewhere)